inst/private/doctest_collect.m

```diff
@@ -64,16 +64,30 @@
   files = dir('.');
   for i=1:numel(files)
     f = files(i).name;
-    if strcmp(f, '.') || strcmp(f, '..') || strcmpi(f, 'private')
-      % skip ., .., and private folders (TODO)
-      continue
-    end
-    if (f(1) == '@')
-      % strip the @, prevents processing as a directory
-      f = f(2:end);
-    elseif (~ recursive && exist(f, 'dir'))
-      % skip directories
-      continue
+    if (exist(f, 'dir'))
+      if (strcmp(f, '.') || strcmp(f, '..'))
+        % skip "." and ".."
+        continue
+      elseif (strcmp(f(1), '@'))
+        % strip @ to prevent processing as a directory
+        f = f(2:end);
```

Comment:
It might!  I've not changed that behaviour with this commit though.  Filed #120.